### ISSUE TITLE
Add automatic account creation

### ISSUE THREAD

**Issue by lhark** [rezometz/paiji2#45]
Apparently the current inscription process discourages most of people to use paiji (as can be seen in the poll results : 50 votes max). It would be a good idea to automaticaly create account for everyone in the residence, and then just send the verification email for them to click. We can then provide a way to change the user id.

**Comment by Paul-FM-Muller** [comment]
Yes, I must admit, as the newcomer, that I found the experience of
subscribing very discomforting. Having to send a mail to the rézo, then
waiting, and the process was, as far as I thought back then, too
complicated for a subscription.
Now, let me learn Django :)

2015-02-05 10:39 GMT+01:00 lhark <notifications@github.com>:

**Comment by liusiqi43** [comment]
Well... the best solution is to start integrating rezo.rez into paiji.rez as the code of rezo.rez is unmaintainable. That will solve a lot of problems and frankly it shouldn't be that much work to rewrite the interface. 

Also the authentication and permission by group is already there in django. So why don't we start to build rezo.rez in django and make that our goal for rezo 2015? 

The ultimate goal is to have one single account across all rezo services.
